pkg/backend/backend_test.go

```diff
@@ -1,26 +1,114 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright (c) 2022 Dell Inc, or its subsidiaries.
 
-package server
+// Package backend implememnts the BackEnd APIs (network facing) of the storage Server
+package backend
 
 import (
 	"bytes"
+	"context"
 	"fmt"
 	"log"
 	"net"
+	"os"
 	"reflect"
+	"strings"
 	"testing"
 
+	"google.golang.org/protobuf/proto"
```

Comment:
nit, but I'd organize this with the other protobuf import below

Reply:
I'll run `goimport` tool to auto-sort this